Document oneagent as the preferred endpoint (#13)
dynatrace-oss/opentelemetry-metric-js: README.md

// configure API endpoint and authentication token

> const exporter = new DynatraceMetricExporter({

Reviewer: Please also update https://github.com/dynatrace-oss/opentelemetry-metric-js/tree/main/samples to allow running it with no URL and token.

  // This is only required if the OneAgent is not installed on the same host

>   // If there is a OneAgent, the exporter will use it to export metrics with no

Reviewer: ```suggestion
  // If there is a OneAgent, by default the exporter will attempt to use it to export metrics with no
```

  // further configuration required.

>   // url: 'https://myenv123.live.dynatrace.com/api/v2/metrics/ingest',
>   // APIToken: 'token123'’,

Reviewer: ```suggestion
  prefix: 'MyPrefix', // optional
  defaultDimensions: [{ // optional
    key: "default-dimension",
    value: "with-value"
  }]
  // If no OneAgent is available locally, export directly to the Dynatrace server:
  // url: 'https://myenv123.live.dynatrace.com/api/v2/metrics/ingest',
  // APIToken: '<load API token from secure location such as env or config file>'’
```
I think leaving prefix and defaultDimensions in makes sense.

// configure API endpoint and authentication token
const exporter = new DynatraceMetricExporter({
  prefix: 'MyPrefix', // optional
  defaultDimensions: [{ // optional
    key: "default-dimension",
    value: "with-value"
  }]

  // If no OneAgent is available locally, export directly to the Dynatrace server:
  // url: 'https://myenv123.live.dynatrace.com/api/v2/metrics/ingest',

>   // APIToken: '<load API token from secure location such as env or config file>'’

Reviewer: ```suggestion
  // APIToken: '<load API token from secure location such as env or config file>'
```

#### Dynatrace API Endpoint

API Endpoint and Token are optional. By default, metrics will be exported to

> the local OneAgent endpoint described below.

Reviewer: ```suggestion
the local OneAgent endpoint described below, if it is available.
```
Since we handle this gracefully.
